modules/administration-guide/partials/assembly_backups-of-che-instances-to-a-rest-backup-server.adoc

```diff
@@ -0,0 +1,11 @@
+[id="backing-up-{prod-id-short}-instances-to-a-rest-backup-server_{context}"]
+= Backups of {prod-short} instances to a REST backup server
+
+You can manage backups of {prod-short} instances to a REST backup server with custom resources or {prod-cli}:
```

Comment:
Here, the meaning of "with" is ambiguous: It can mean "by using" but can also be misunderstood as meaning that the backup server *has* custom resources. This type of semantic noise affects users who are less experienced with the technology or whose first language is not English. 
```suggestion
You can manage backups of {prod-short} instances to a REST backup server by using custom resources or {prod-cli}:
```

Reply:
Would you consider making similar updates elsewhere in this PR?

Reply:
@rolfedh Thank you, Rolfe. I've fixed this across this PR.